<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fereastra_principala</class>
 <widget class="QWidget" name="fereastra_principala">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>medii - activatate</string>
  </property>
  <widget class="QPushButton" name="buton_seminar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton_laborator">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Laborator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
